src/app/components/core/user-management/update-user/update-user.component.ts

```diff
@@ -28,38 +31,41 @@ export class UpdateUserComponent implements OnInit, OnChanges {
     this.userRecord = {
       userId: 0,
       username: '',
+      description: '',
       password: '',
+      real_name: '',
+      access_method: '',
       confirmPassword: '',
       role_id: 0   // set "user" as a default role
     };
+    this.loginMethods = [{text: 'Any', value: 'any'}, {text: 'Password', value: 'pwd'}, {text: 'Certificate', value: 'cert'}];
```

Comment:
can now we reuse same dict as in another file? better we use same for display in config page LATER